code/parachain/runtime/dali/src/xcmp.rs

```diff
@@ -103,22 +108,52 @@ pub type XcmOriginToTransactDispatchOrigin = (
 
 pub struct StaticAssetsMap;
 
-pub mod parachains {
-	pub mod karura {
-		pub const ID: u32 = 3000;
-		pub const KUSD_KEY: &[u8] = &[0, 129];
-	}
-}
-
 impl XcmpAssets for StaticAssetsMap {
 	fn remote_to_local(location: MultiLocation) -> Option<CurrencyId> {
 		match location {
 			MultiLocation { parents: 1, interior: X2(Parachain(para_id), GeneralKey(key)) } =>
 				match (para_id, &key[..]) {
-					(parachains::karura::ID, parachains::karura::KUSD_KEY) =>
-						Some(CurrencyId::kUSD),
+					(topology::karura::ID, topology::karura::KUSD_KEY) => Some(CurrencyId::kUSD),
 					_ => None,
 				},
+			MultiLocation {
+				parents: 1,
+				interior: X3(Parachain(para_id), PalletInstance(pallet_instance), GeneralIndex(key)),
+			} => match (para_id, pallet_instance, key) {
+				(
+					topology::common_good_assets::ID,
+					topology::statemine::ASSETS,
+					topology::statemine::USDT,
+				) => Some(CurrencyId::USDT),
+				_ => None,
+			},
+			_ => None,
+		}
+	}
+
+	fn local_to_remote(id: CurrencyId, this_para_id: u32) -> Option<MultiLocation> {
+		match id {
+			CurrencyId::NATIVE => Some(topology::SELF_RECURSIVE),
+			CurrencyId::RELAY_NATIVE => Some(MultiLocation::parent()),
+			CurrencyId::kUSD => Some(MultiLocation {
+				parents: 1,
+				interior: X2(
+					Parachain(topology::karura::ID),
+					GeneralKey(WeakBoundedVec::force_from(
+						topology::karura::KUSD_KEY.to_vec(),
+						None,
+					)),
+				),
+			}),
+			CurrencyId::USDT => Some(MultiLocation {
+				parents: 1,
+				interior: X3(
+					Parachain(topology::common_good_assets::ID),
+					PalletInstance(topology::common_good_assets::ASSETS),
+					GeneralIndex(topology::common_good_assets::USDT),
+				),
+			}),
```

Comment:
Is it possible to store the `MultiLocation`s of these well known assets as constants?

Reply:
you mean reuse? yes. can reuse. I do not know where. In test it seems viable to type all over again so tests are visual. at least for now. but yes - can make constants.

Reply:
Also used in `code/parachain/runtime/picasso/src/xcmp.rs:153`

Reply:
fixing this

Reply:
fixed, but durign DOT <-> KSM bridge it will be broken and no 100% reuse.